src/sinol_make/util.py

```diff
@@ -74,6 +80,13 @@ def install_oiejq():
     if not os.path.exists(os.path.expanduser('~/.local/bin')):
         os.makedirs(os.path.expanduser('~/.local/bin'), exist_ok=True)
 
+    if os.path.exists(os.path.expanduser('~/.local/bin/oiejq')) and \
+            not _check_if_oiejq_executable(os.path.expanduser('~/.local/bin/oiejq')):
```

Comment:
Isn't it enough to check only the second condition?

Reply:
It isn't, because `_check_if_oiejq_executable(path)` returns `False` if the `path` doesn't exist